samples/javascript_nodejs/56.teams-file-upload/services/fileService.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const util = require('util');
+const axios = require('axios');
+const readdir = util.promisify(fs.readdir);
+
+// Generates File Name with Sequence
+const geneFileName = async (fileDir) => {
+    const filenameConst = 'UserAttachment';
+    const files = await readdir(fileDir);
+    const filteredFiles = files.filter(f => f.includes(filenameConst)).map(f => parseInt(f.split(filenameConst)[1].split('.')[0]));
+    const maxSeq = Math.max(0, filteredFiles);
+    const filename = `${ filenameConst }${ maxSeq + 1 }.png`;
+    return filename;
+};
+
+// Download and Save Streams into File
+const response = await axios({ method: 'GET', url: contentUrl, responseType: 'stream' });
```

Comment:
Sorry - this should still be inside a function `writeFile` 😄

Reply:
yes, sorry missed it, added now, you can check it.